lista/30/renan.js

```diff
@@ -0,0 +1,39 @@
+// 30) [DESAFIO] Refaça o algoritmo 25, acrescentando o recurso de mostrar que tipo
+// de triângulo será formado:
+//  - EQUILÁTERO: todos os lados iguais
+//  - ISÓSCELES: dois lados iguais
+//  - ESCALENO: todos os lados diferentes
```

Comment:
Dica, tem como deixar esse código com umas 17 linhas a menos.

É triangulo ou não é booleano e vc pode usar isso direto nas condicionais, e dentro do corpo das condicionais avaliar o tipo do triangulo, se não for triangulo vc diz que a mãe do cara é puta e aquilo n da um triangulo